src/app/portlets/content-types/fields/content-type-fields-add-row/content-type-fields-add-row.component.ts

```diff
@@ -0,0 +1,162 @@
+import { BaseComponent } from './../../../../view/components/_common/_base/base-component';
+import { Component, OnInit, OnDestroy, Output, EventEmitter, ViewChild, ElementRef } from '@angular/core';
+import { FieldRow } from '../shared';
+import { HotkeysService, Hotkey } from 'angular2-hotkeys';
+import { trigger, state, style, transition, animate } from '@angular/animations';
+import { MessageService } from './../../../../api/services/messages-service';
+
+/**
+ * Display select columns row
+ *
+ * @export
+ * @class ContentTypeFieldsAddRowComponent
+ */
+@Component({
+    selector: 'content-type-fields-add-row',
+    animations: [
+        trigger('enterAnimation', [
+            state(
+                'expanded',
+                style({
+                    height: '*',
+                    overflow: 'visible'
+                })
+            ),
+            state(
+                'collapsed',
+                style({
+                    height: '0px',
+                    overflow: 'hidden'
+                })
+            ),
+            transition('expanded <=> collapsed', animate('300ms ease-in-out'))
+        ])
+    ],
+    styleUrls: ['./content-type-fields-add-row.component.scss'],
+    templateUrl: './content-type-fields-add-row.component.html',
+})
+export class ContentTypeFieldsAddRowComponent extends BaseComponent implements OnInit, OnDestroy {
+    rows: number[] = [1, 2, 3, 4];
+    row: number;
+    fieldRows: FieldRow[];
+    addRowSelected = 'collapsed';
+    selectedCol = 0;
+    disabledButton = false;
+    comboHotKeys: Hotkey;
```

Comment:
I think you don't need this attribute, because yo can get the HotKey from the HotkeysService